### DISCUSSION TITLE
Leveling Automated Passive Tree from Export

### DISCUSSION THREAD

**Discussion by bearryj** [Lailloken/Exile-UI#262] [Ideas & Feedback]
Hi, I've been using the legacy UI for the past couple of days getting ready for the league. I was wondering if the new 1.50 update will have automated passive skill trees on export from exile leveling since it already tracks that data. Thanks!

**Comment by Lailloken** [comment]
It was pure coincidence that we both implemented a skill tree feature. I chose an easier and simpler solution, so it was ready earlier. We sometimes chat about dev stuff, and I could tell their implementation took a whole lot of work. So it didn't feel right to ask if the skill tree could also be included in the export: "Yo, I didn't feel like putting in a fancy solution because it would have been too much work. But now that I see yours, I would like to have that as well. Gimme."

That being said, the skill tree import and management has been streamlined a bit for v1.50, and you can use it to capture the exile leveling skill trees as well (captions for masteries would need to be added manually, though).

**Reply by bearryj** [reply]
Aren't they just basically taking the data from PoB.

**Reply by Lailloken** [reply]
The data, yes, but the skill tree schematic had to be built for that, as far as I understand (and it sounded frustrating).
